webapp/src/ts/services/enketo.service.ts

```diff
@@ -288,44 +289,65 @@ export class EnketoService {
       });
   }
 
-  private renderFromXmls(doc, wrapper, instanceData) {
+  private renderFromXmls(xmlFormContext: XmlFormContext) {
+    const { wrapper } = xmlFormContext;
     wrapper
       .find('.form-footer')
       .addClass('end')
       .find('.previous-page,.next-page')
       .addClass('disabled');
 
     const formContainer = wrapper.find('.container').first();
-    formContainer.html(doc.html);
+    formContainer.html(xmlFormContext.doc.html.get(0));
```

Comment:
do we need to null check html?
`xmlFormContext.doc.html?.get(0)`

Was it failing before because we didnt do `.get(0)`?

Reply:
We shouldn't need to null check html. Typescript was throwing an error here because the argument we were passing to `formContainer.html(...)` was not valid.  
The old code would work thanks to jQuery being lenient and backwards-compatible but their documentation matches the types we have so I updated the code.